.github/workflows/build_azure_pipelinesrelease_template.yml

```diff
@@ -78,7 +78,7 @@ jobs:
   Validate-unit_test:
     name: Unit Test
     runs-on: ubuntu-latest
-    if: !(inputs.skipTests)
+    if: ${{inputs.skipTests == false}}
```

Comment:
Only thing I would say is if we are moving from Azure pipelines, maybe let's rename the  file?